packages/loader/container-loader/src/container.ts

```diff
@@ -1769,4 +1781,60 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private logContainerError(warning: ContainerWarning) {
         this.logger.sendErrorEvent({ eventName: "ContainerWarning" }, warning);
     }
+
+    private async loadContainerCopy(
+        // back-compat 0.35 stop passing loader post 0.35 support
+        loader: Loader,
+        urlResolver: IUrlResolver,
+        baseRequestUrl: string | undefined,
+        additionalHeaders: IRequestHeader,
```

Comment:
Can this take an `IContainerLoadOptions` (or maybe some scoped subset of it) instead of an `IRequestHeader` perhaps?